WordPress/src/main/java/org/wordpress/android/ui/whatsnew/FeatureAnnouncementProvider.kt

```diff
@@ -1,37 +1,47 @@
 package org.wordpress.android.ui.whatsnew
 
-import org.wordpress.android.R
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import org.wordpress.android.WordPress
+import org.wordpress.android.util.AppLog
+import org.wordpress.android.util.AppLog.T
+import java.io.FileNotFoundException
 import javax.inject.Inject
 
 class FeatureAnnouncementProvider @Inject constructor() {
-    private val localFeatureAnnouncement = FeatureAnnouncement(
-            "14.7", 857, "https://wordpress.com/blog/2020/04/20/earth-day-live/", listOf(
-            FeatureAnnouncementItem(
-                    "Super Publishing",
-                    "Publish amazing articles using the power of your mind! Concentrate" +
-                            " on what you want to post, and we will do the rest!",
-                    R.drawable.ic_posts_white_24dp
-            ),
-            FeatureAnnouncementItem(
-                    "Great cats and superb dogs are right behind you!",
-                    "That's right! They are right in the app! They require pets right now." +
-                            " Are you going to look for them or what?",
-                    R.drawable.ic_plans_white_24dp
-            ),
-            FeatureAnnouncementItem(
-                    "We like long feature announcements that why this one is going to be extra" +
-                            " long and span multiple lines",
-                    "Here we run out of budget.",
-                    R.drawable.ic_themes_white_24dp
-            )
-    )
-    )
+    private val gson: Gson by lazy {
+        val builder = GsonBuilder()
+        builder.create()
+    }
 
     fun getLatestFeatureAnnouncement(): FeatureAnnouncement? {
-        return localFeatureAnnouncement
+        return getFeatureAnnouncements().firstOrNull()
+    }
+
+    fun getFeatureAnnouncements(): List<FeatureAnnouncement> {
+        val featureAnnouncements = arrayListOf<FeatureAnnouncement>()
+
+        var featureAnnouncementFileContent: String? = null
+        try {
+            featureAnnouncementFileContent = WordPress.getContext().assets
+                    .open("FEATURE_ANNOUNCEMENTS.json")
```

Comment:
Looking here I was think that maybe can come handy to place a sort of json template file like we do for `gradle.properties-example` and placing a sort of empty state instead of removing the file like

```
{
  "announcements": [
    {
    }
  ]
}
```

No need to decide in this PR (we have few others upcoming so we can better think how to manage this), but wanted to mention.

PS: out of curiosity, is there a reason for the all capital letters?

Reply:
Good idea! I'll add it in subsequent PR since I also wanted to work on making some elements optional. I can reflect those rules in example file.

All caps to mimic RELEASE-NOTES.txt. It doesn't have to be like this tho :)